android/app/src/main/java/poke/rogue/helper/presentation/dex/PoketmonViewHolder.kt

```diff
@@ -3,20 +3,39 @@ package poke.rogue.helper.presentation.dex
 import androidx.recyclerview.widget.RecyclerView
 import poke.rogue.helper.databinding.ItemPokemonListPokemonBinding
 import poke.rogue.helper.presentation.dex.model.PokemonUiModel
+import poke.rogue.helper.presentation.type.view.PokemonTypeViewConfiguration
 
 class PoketmonViewHolder(
     private val binding: ItemPokemonListPokemonBinding,
-    private val onClickPokeMonItem: PokemonListNavigateHandler,
+    onClickPokeMonItem: PokemonListNavigateHandler,
 ) : RecyclerView.ViewHolder(binding.root) {
     init {
         binding.listener = onClickPokeMonItem
     }
 
     fun bind(pokemonUiModel: PokemonUiModel) {
         binding.pokemon = pokemonUiModel
-        binding.rvPokeTypeList.adapter =
-            PokemonTypeAdapter().apply {
-                submitList(pokemonUiModel.types)
-            }
+        val typesLayout = binding.layoutItemPokemonPokemonTypes
+
+        val pokemonTypesAdapter =
+            PokemonTypesAdapter(
+                context = binding.root.context,
+                viewGroup = typesLayout,
+            )
+
+        pokemonTypesAdapter.addTypes(
+            types = pokemonUiModel.types,
+            config = TypesUiConfig,
+        )
+    }
+
+    companion object {
+        private val TypesUiConfig =
+            PokemonTypeViewConfiguration(
+                isEmptyBackground = false,
+                nameSize = 10f,
+                iconSize = 20f,
+                marginBetweenTypes = 0,
```

Comment:
리마인드 : `dp` util 함수 넣어줘야하는거 잊지 말기!

spacing 으로 네이밍 통일화하는거 어떠신가요?
```suggestion
                nameSize = 10f,
                iconSize = 20f,
                spacing = 0,
```